### PR TITLE
added crossGroupRunAlignment and fixed mainGroupAlignment.

### PR DESCRIPTION
**Pull request by AthreshK**
Hi, The package is really helpful. 
I wanted to achieve a styling where the buttons are center-aligned when horizontally placed like this
drawing

But the maximum i could achive was this
drawing

It turns out that the property _alignment_ is actually responsible for the main axis alignment instead of the _runAlignment_ which was used previously.
But _runAlignment_ is also useful to achive somthing like  
drawing 

So I corrected the main axis alignment property and added the run alignment too so that all 3 properties can be customized for the best styling.

What do you think? Hope this helps! 😄

### CODE REVIEW

File: lib/src/utils/alignments.dart [hunk 1]
```diff
@@ -14,3 +14,13 @@ enum CrossGroupAlignment {
   center,
   end,
 }
+
+/// How the button runs themselves should be placed the cross axis in a layout
+enum CrossGroupRunAlignment {
```
Comment (Frezyx) [inline lib/src/utils/alignments.dart:19]: Please refactor this name **CrossGroupRunAlignment** -> **GroupRunAlignment**

**Review by Frezyx — CHANGES_REQUESTED** [review]
Hello @AthreshK ✋🏻 
Thank you very much for this changes !

Please complete review tasks and I merge this code to new version

File: lib/src/utils/alignments.dart (hunk 1 above)
Comment (AthreshK) [reply lib/src/utils/alignments.dart]: Done with the refactoring. Thank you, Waiting for the version release! 🙌

**Review by Frezyx — APPROVED** [review]

**Comment by Frezyx** [comment]
Thank you !
You can see this changes un new packages version

**Pull request merged**